docs/_posts/2018-01-15-jekyll-3-7-1-released.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Jekyll 3.7.1 Released"
+date: 2018-01-15 11:21:00 +0530
+author: ashmaroli
+version: 3.7.1
+categories: [release]
+---
+
+Close on the heels of shipping 3.7.0, we were informed of a couple of
+regressions due to the changes made in that release. In due time, Team Jekyll
+set out to address those issues as early as possible.
+
+Days later, here we're announcing 3.7.1 that aims to set things in order. The
+highlights being:
+
+  * A major regression in 3.7.0 was that when a Front Matter Default was
+    configured with a `scope["path"]` set to a directory, Jekyll would scan
+    that directory for any subfolders and files, for each document in that
+    `path`.
+    Though this is intended, it increases build times in proportion to the size
+    of the said directory. <br />
+    We addressed this by having Jekyll scan the directory path only if the user
+    explicitly configures the `scope["path"]` using wildcards. <br />
+    Read our documentation for more details. <br />
+    A huge shout-out to @mmistakes for bringing this to our notice and
+    additionally providing with a test repository to aid in resolving the issue.
+  * Another regression reported was related to our "Custom collections
+    directory" feature introduced in 3.7.0. <br />
+    Users setting `collection_dir` to a certain directory would have *altered*
+    paths to their posts still at the root of their site's source. This
+    roughly translated to 404 errors on URLs to their posts. <br />
+    Props to @localheinz for asserting this regression to our notice. <br />
+    We decided to resolve this by having Jekyll ignore posts and drafts at the
+    root of the site's source directory *if the user customizes the
+    `collection_dir` setting.* <br />
+    Ergo, if you set a custom location for your collections, please ensure you
+    move all of your collections into that directory. **This includes posts and
+    drafts as well**. Your links generated by
+    `{% raw %}{% post_url %}{% endraw %}` or `{% raw %}{% link %}{% endraw %}`
+    will adapt automatically.
+
+In addition to the above, numerous other minor fixes and documentation updates
+have been made that should improve your Jekyll experience. All of which would
+not have been possible without our wonderful contributors: Andreas Möller,
+Ashwin Maroli, Florian Thomas, Frank Taillandier, Olivia and Parker Moore.
+
```

Comment:
Update to : Alexandr, Andreas Möller, Ashwin Maroli, Florian Thomas, Frank Taillandier, Kacper Duras, Olivia, Parker Moore, Paul Robert Lloyd